controllers/memberoperatorconfig/cache_test.go

```diff
@@ -104,12 +114,185 @@ func TestGetConfigFailed(t *testing.T) {
 	})
 }
 
+func TestLoadLatest(t *testing.T) {
+	t.Run("config found", func(t *testing.T) {
+		initconfig := NewMemberOperatorConfigWithReset(t, testconfig.MemberStatus().RefreshPeriod("1s"))
+		// given
+		cl := NewFakeClient(t, initconfig)
+
+		// when
+		err := loadLatest(cl, MemberOperatorNs)
+
+		// then
+		require.NoError(t, err)
+	})
```

Comment:
it's probably not a big deal, but you don't verify the set/updated values in the cache. 
I know that you did it for the `updateConfig` function a few lines above, but that is a different function.

Reply:
yeah, that's a fair point. GetConfig and loadLatest are used in different cases so it's worth verifying in both. Added tests for when the config has and hasn't changed in cabf09c52d96346671d13a4c55a314bc3e08f667